src/SFA.DAS.EAS.Employer_Financial.Database/StoredProcedures/DeleteSubmissions_ByEmpRef.sql

```diff
@@ -13,14 +13,14 @@ AS
 
 	DELETE	
 	FROM	employer_financial.TransactionLine 
-	WHERE	SubmissionId IN (SELECT SubmissionId FROM @SubmissionIdsInUseByEmpRef);
+	WHERE EmpRef = @empRef
 
 	DELETE 
 	FROM	employer_financial.LevyDeclarationTopup
 	WHERE	SubmissionId IN (SELECT SubmissionId FROM @SubmissionIdsInUseByEmpRef);
```

Comment:
We could select directly from the LevyDeclaration table now, and remove the table variable, now that we only need the results in one place.

Reply:
I've reverted the sproc to how it used to work to ensure it doesn't break the other tests and just added `OR EmpRef = @empRef`.